README.md

```diff
@@ -16,7 +16,7 @@ Along with these ML capabilities, this first release of ML.NET also brings the f
 
 [![NuGet Status](https://img.shields.io/nuget/v/Microsoft.ML.svg?style=flat)](https://www.nuget.org/packages/Microsoft.ML/)
 
-ML.NET runs on Windows, Linux, and macOS - any platform where 64 bit [.NET Core](https://github.com/dotnet/core) or later is available.
+ML.NET runs on Windows, Linux, and macOS - any platform where x64 bits [.NET Core](https://github.com/dotnet/core) or later is available.
```

Comment:
If we use `x64`, I believe we can drop the `bits`, e.g.:

```
ML.NET runs on Windows, Linux, and macOS - any platform where x64 [.NET Core](https://github.com/dotnet/core) or later is available.
```

Reply:
Then again, .NET Core is only available on x64, not on x86. .NET Framework is available for both. This whole statement might need a rewording, as we support 

  * some of ML.NET on Windows on .NET Framework on x86 (soon),
  * all of it on .NET Framework and Core on Windows on x64, and
  * all of it on .NET Core on Windows, Linux and macOS on x64

Reply:
> .NET Core is only available on x64, not on x86

This is not correct. .NET Core is available for and ships for both x86 and x64.

Reply:
I believe the list should be
* some of ML.NET on Windows on .NET Framework and .NET Core on x86 (soon),
* all of it on .NET Framework and Core on Windows on x64, and
* all of it on .NET Core on Windows, Linux and macOS on x64
* preview level of some of ML.NET on Linux on ARM32 (not sure about ARM64 @tannergooding ?)

Reply:
Therefore, the following statement would be true until we don't support x86 or ARM, right?
_ML.NET runs on Windows, Linux, and macOS - any platform where x64 [.NET Core](https://github.com/dotnet/core) or later is available. In addition, .NET Framework on Windows x64 is also supported._

Reply:
OK fine @artidoro when you have CI running for x86 Windows please add it to this text...